Cargo.toml

```diff
@@ -1,9 +1,10 @@
 [package]
 name = "tomlq"
-version = "0.1.6"
+version = "0.2.0"
```

Comment:
Please rebase off of the latest main to drop the commits that aren't connected to this PR